Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6930,6 +6928,89 @@ def initialize_instance(args: Dict[str, str], params: Dict[str, str]):
         XPATH_RULEBASE = f"/config/devices/entry[@name=\'localhost.localdomain\']/vsys/entry[@name=\'{VSYS}\']/"
 
 
+def panorama_upload_content_update_file_command(args: dict):
+    category = args.get('category')
+    entry_id = args.get('entryID')
+    file_path = demisto.getFilePath(entry_id)['path']
+    file_name = demisto.getFilePath(entry_id)['name']
+
+    try:
+        shutil.copy(file_path, file_name)
+    except Exception:
+        raise Exception('Failed to prepare file for upload.')
+
+    with open(file_name, 'rb') as file:
+        files = {'file': file}
+        params = {'type': 'import', 'category': category, 'key': API_KEY}
+        response = http_request(uri=URL, method="POST", headers={}, body={}, params=params, files=files)
+        human_readble = tableToMarkdown("Results", t=response.get('response'))
+        content_upload_info = {
+            'Message': response['response']['msg'],
+            'Status': response['response']['@status']
+        }
+        results = CommandResults(raw_response=response,
+                                 readable_output=human_readble,
+                                 outputs_prefix="Panorama.Content.Upload",
+                                 outputs_key_field="Status",
+                                 outputs=content_upload_info)
+
+    shutil.rmtree(file_name, ignore_errors=True)
+    return results
+
+
+@logger
+def panorama_install_file_content_update(version: str, category: str, validity: str):
```

Comment:
Adding a link to the API in a docstring could be helpful here, if possible

Reply:
Done